pages/index.tsx

```diff
@@ -65,7 +65,7 @@ const RatingForkComponent: React.FC<RatingForkProps> = ({
 
   return (
     <div
-      className={`dark:text-white rounded-lg md:w-[160px] text-3xl p-4 dark:bg-[rgba(255,255,255,0.1)] bg-[rgba(0,0,0,0.05)] w-full`}
+      className={`dark:text-white rounded-lg w-[200px] text-3xl p-4 dark:bg-[rgba(255,255,255,0.1)] bg-[rgba(0,0,0,0.05)] w-full`}
```

Comment:
Hey please add `md` as `md` represents to medium screen size, by removing it, its setting width to `200px` for all screen size.

Reply:
@rupali-codes  

Yes. Will fix it.